motelpos/MotelPos/Logic/Extensions/toString.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  Date+String.swift
+//  MotelPos
+//
+//  Created by Amir Kamali on 6/6/18.
+//  Copyright © 2018 Assembly Payments. All rights reserved.
+//
+
+import Foundation
+
+extension Date {
+    
+    func toString(includeTime: Bool = false) -> String {
+        var format = ""
+        if (includeTime) {
+            format = "dd/MM/yyyy hh:mm a"
+        } else {
+            format = "dd/MM/yyyy"
+        }
+        return toString(format: format)
+    }
+    
+    func toString(format: String) -> String {
+        let dateFormatter = DateFormatter()
```

Comment:
Creating new DateFormatter is really expensive and slow, just a heads up - if this is called iterating over heaps of rows that'd be a problem. If used once per request then no problem.